src/coreComponents/common/PhysicsConstants.hpp

```diff
@@ -0,0 +1,113 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file PhysicsConstants.hpp
+ * @brief Regroups useful constants and functions relative to Units Of Measures
+ * that are globally used for math and physics computations.
+ */
+#ifndef GEOS_MATH_PHYSICSCONSTANTS_HPP_
+#define GEOS_MATH_PHYSICSCONSTANTS_HPP_
+
+#include "common/DataTypes.hpp"
+
+namespace geos
+{
+
+
+namespace constants
+{
+
+/// @brief Zero degree Celsius in Kelvin
+constexpr double zeroDegreesCelsiusInKelvin = 273.15;
+
+}
+
+
+namespace units
```

Comment:
At some moments, we considered using a C++ `units_lib` library like
```c++
#ifdef ENABLE_UNITS
...
using Celcius = units_lib::temperature< double, ...>;
using Kelvin = units_lib::temperature< double, ...>;
...
#else
...
using Celcius = double;
using Kelvin = double;
...
#endif
```
We don't have to run it on GPU since it's only validation, it can be done once on the `CI` (e.g.).
Do you think your use case could be a good opportunity for the introduction of such a thing?

Reply:
I think that would be an excellent idea (I'm planning to add a similar library), but I think this is too much for this PR.
I would like to add a specific PR for adding a library and wrapper types, and another PR to expose tools to the user to choose the units he wants to work with.

Reply:
We used to have a unit conversion feature. It should be done on reading of XML, and setting of default values, and then at writing to output. Never inside the code.